recordings-downloader/nodejs/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "nodejs",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC"
```

Comment:
License needs to be MIT and the package needs to have dependencies specified. You should be able run `npm i purecloud-platform-client-v2` in this directory and it'll add it to the file for you.

Reply:
done changing the package.json license and dependencies.